static/js/vue-cdr-access/src/components/searchWrapper.vue

```diff
@@ -108,12 +115,20 @@ Top level component wrapper for search pages
                     this.filter_parameters = response.data.filterParameters;
                     this.min_created_year = response.data.minSearchYear;
                     this.is_loading = false;
+                    this.query_changed = false;
                 }).catch(function (error) {
                     console.log(error);
+                    this.query_changed = false;
                 });
             }
         },
 
+        // check if query params other than "start" have changed otherwise pagination can't get past the first page
+        beforeRouteUpdate(to, from) {
+            this.query_changed = !isEqual(this.removeQueryParameters(['start'], to.query),
```

Comment:
I'm confused how this is working:
`this.removeQueryParameters(['start'], to.query)`
The function seems to accept an array and a boolean, i don't see a version of it that would remove `['start']` from the provided `query` object, am I missing something?